modules/nf-core/cnvnator/convert2vcf/main.nf

```diff
@@ -0,0 +1,47 @@
+process CNVNATOR_CONVERT2VCF {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "bioconda::cnvnator=0.4.1"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/cnvnator:0.4.1--py310h2dce045_7':
+        'biocontainers/cnvnator:0.4.1--py310h2dce045_7' }"
+
+    input:
+    tuple val(meta), path(calls)
+
+    output:
+    tuple val(meta), path("*.vcf"), emit: vcf
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args       = task.ext.args   ?: ''
+    def prefix     = task.ext.prefix ?: "${meta.id}"
+    def output_cmd = "> ${prefix}.vcf"
```

Comment:
maybe we should move this to the cnvnator command since there is no check or options anymore. 
```suggestion
```